test/jdk/java/foreign/TestLayouts.java

```diff
@@ -284,6 +284,25 @@ public void testAlignmentString(MemoryLayout layout, long bitAlign) {
         }
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testSequenceElement() {
+        SequenceLayout layout = MemoryLayout.sequenceLayout(10, JAVA_INT);
+        // Step must be != 0
+        PathElement.sequenceElement(3, 0);
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void test() {
```

Comment:
I suggest to add some name - e.g. `testBadValueCarrier`

Reply:
Ahh. I forgot to change the temporary name. Thanks for spotting this.